src/seed/seed_db.ts

```diff
@@ -19,6 +20,14 @@ const logger: Logger = _logger("seed_db");
 const databaseDownloadDir = process.env.AOIMIRAI_DUMP_DIR;
 const overridesFilePath = path.join(__dirname, "./kpop_videos_overrides.sql");
 
+program
+    .option("-fd, --force-download", "Force re-download of AoiMirai backup", false)
+    .option("-fr, --force-reseed", "Force drop/create of kpop_videos database", false)
+    .option("-sd, --skip-download", "Skip download/encode of videos in database", false);
```

Comment:
force-download and skip-download might seem related at first glance, but they deal with different parts of the seeding process, if you don't mind can you rename skip-download?